Libplanet.Net/Messages/ConsensusCommit.cs

```diff
@@ -1,19 +1,35 @@
-using Libplanet.Blocks;
+using System.Collections.Generic;
+using Libplanet.Consensus;
 
 namespace Libplanet.Net.Messages
 {
     public class ConsensusCommit : ConsensusMessage
     {
-         public ConsensusCommit(long nodeId, long height, long round, BlockHash blockHash)
-         : base(nodeId, height, round, blockHash)
-         {
-         }
+        public ConsensusCommit(Vote vote)
+            : base(vote.NodeId, vote.Height, vote.Round, vote.BlockHash)
+        {
+            CommitVote = vote;
+        }
 
-         public ConsensusCommit(byte[][] dataframes)
-         : base(dataframes)
-         {
-         }
+        public ConsensusCommit(byte[][] dataframes)
+            : this(new Vote(dataframes[0]))
+        {
+        }
 
-         public override MessageType Type => MessageType.ConsensusCommit;
+        public Vote CommitVote { get; }
+
+        public override IEnumerable<byte[]> DataFrames
+        {
+            get
+            {
+                var frames = new List<byte[]>
+                {
+                    CommitVote.ByteArray,
+                };
+                return frames;
```

Comment:
May be missing `base.DataFrames`?

Reply:
https://github.com/planetarium/libplanet/pull/1979/files#diff-8f7496d7becdca3f0362db3cbd3c09f4c0cc16923436b0f4aeec5e094be984bcR14-R15 You should check out here. `base.DataFrames` is unnecessary. it initialized on the constructor.